cell/cell.go

```diff
@@ -23,8 +23,12 @@ type Option interface {
 
 // Options stores the provided options.
 type Options struct {
-	FgColor Color
-	BgColor Color
+	FgColor       Color
```

Comment:
Would you be also willing to add test coverage for the modified modules? There are pre-existing unit test that should provide some guidance.

Reply:
Sure thing!